sktm/executable.py

```diff
@@ -205,6 +210,12 @@ def load_config(args):
     else:
         cfg['report_footer'] = os.path.abspath(cfg['report_footer'])
 
+    jretry = cfg.get('jretry')
```

Comment:
There's no need for this - `if not cfg.get('jretry')` and `cfg['jretry'] = int(cfg.get('jretry'))` should work just fine :)

Reply:
Agree, updated.